keras_nlp/models/distilbert/distilbert_models.py

```diff
@@ -43,6 +43,11 @@ class DistilBert(keras.Model):
     dimensions. To load preset architectures and weights, use the `from_presets`
     constructor.
 
+    Disclaimer: Pre-trained models are provided on an "as is" basis, without
+    warranties or conditions of any kind. The underlying model is provided by a
+    third party and subject to a separate license, found at:
+    https://github.com/huggingface/transformers
```

Comment:
We're going to be rendering this markdown on the website, so please format the links as markdown links, e.g. `available [here](https://github.com/huggingface/transformers)`